### ISSUE TITLE
Support for Orange Pi 5 Plus

### ISSUE THREAD

**Issue by b9AcE** [pyavitz/debian-image-builder#102]
Having noted that the "Orange Pi 5"-device is already at least somewhat supported (work in progress), I wonder what would be needed to also add support for the similar "[Orange Pi 5 Plus](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-5-plus.html)"?
It has e.g. several Debian images officially from the device manufacturer (bookworm and bullseye with 6.1.x and 5.10.x kernels) as well as several other device manufacturer official and third-party Linux [distro-images](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/service-and-support/Orange-Pi-5-plus.html), which could aid the adding of the device to this script.
Hoping to be able to use Devuan on the device through this method.

**Comment by pyavitz** [comment]
It builds. I have no clue if or how it works. https://github.com/pyavitz/debian-image-builder/pull/103

```
ls *orangepi-5-*
rk3588-orangepi-5-plus-devuan-daedalus-5.10.110-arm64-ext4-2024-03-02-1148.img
```

**Comment by b9AcE** [comment]
I built it (on the Orange Pi 5 Plus running DietPi) with no apparent issues, then flashed it onto an SD card using balenaEtcher, then booted the Orange Pi 5 Plus from that and it all seems to work fine, including both wired and wireless networking as far as I tested it. The NVMe  on the integrated M.2 at least showed up to lsblk but I did not test that any farther.
I have not tested other devices such as Bluetooth or audio.
Thank you!
